### PR TITLE
Add new --include-json arg to vellum pull

### PR DESCRIPTION
**Pull request by dvargas92495**
Should help codegen crew generate tons of fixtures for robust testing.

Let me know if we'd like to kick off a new release with this change

### CODE REVIEW

File: ee/vellum_cli/__init__.py [hunk 1]
```diff
@@ -43,9 +43,10 @@ def push(
 @main.command()
 @click.argument("module", required=False)
 @click.option("--legacy-module", is_flag=True, help="Pull the workflow as a legacy module")
-def pull(module: Optional[str], legacy_module: Optional[bool]) -> None:
+@click.option("--include-json", is_flag=True, help="Include the workflow JSON in the pull response")
```
Comment (noanflaherty) [inline ee/vellum_cli/__init__.py]:
```suggestion
@click.option("--include-json", is_flag=True, help="Include the JSON representation of the Workflow in the pull response. Should only be used for debugging purposes.")
```

File: ee/vellum_cli/pull.py [hunk 2]
```diff
@@ -70,4 +80,7 @@ def pull_command(module: Optional[str], legacy_module: Optional[bool] = None) ->
                 logger.info(f"Writing to {target_file}...")
                 target.write(source.read().decode("utf-8"))
 
+    if include_json:
+        logger.warning("Workflow JSON included should be considered unstable and subject to change")
```
Comment (noanflaherty) [inline ee/vellum_cli/pull.py]:
```suggestion
        logger.warning("The pulled JSON representation of the Workflow should be used for debugging purposely only. Its schema should be considered unstable and subject to change at any time.")
```

**Review by noanflaherty — APPROVED** [review]
🔥

I'd love help cutting a release as this goes in

**Comment by kirawoods** [comment]
❤️

**Pull request merged**